Dockerfile

```diff
@@ -32,4 +33,6 @@ COPY . .
 RUN cd ./frontend && yarn install && wasm-pack build . --target=web --dev
 RUN cargo tauri build --debug && mv /usr/src/myapp/target/debug/tauri /usr/bin/autodox && rm -rf /usr/src/myapp
 RUN  dbus-uuidgen > /etc/machine-id
-CMD ["/usr/bin/autodox"]
+#CMD ["/usr/bin/autodox"]
+CMD cargo tarui dev
+channgroup.usersuserwueruserspassprinttry:ecexcpass
```

Comment:
i don't understand this...

Reply:
never mind it. Just experimenting 
also, the `CMD cargo tarui dev` runs the command directly 
also, don't need `CMD ["/usr/bin/autodox"]` because it does nothing

Reply:
no that will rebuild the tauribapp everytime it is run
we just need the binary

Reply:
aha ok ok, never mind about this branch. But just remove the fedora and use `rust:latiest`